docs/how-to-guides/develop-a-flex-flow/class-based-flow.md

```diff
@@ -37,6 +37,8 @@ class MyFlow:
 
 ## Flow test
 
+### Test with original code
```

Comment:
"original code" is hard to understand for first-use user
Not sure if below is better?
Test via function call